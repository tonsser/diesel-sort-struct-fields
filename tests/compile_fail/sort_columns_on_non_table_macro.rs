#![allow(unused_imports, unused_macros)]

#[macro_use]
extern crate diesel;

use diesel::prelude::*;
use diesel_sort_struct_fields::sort_columns;

macro_rules! foo {
    () => {}
}

#[sort_columns]
foo! {}

fn main() {}
