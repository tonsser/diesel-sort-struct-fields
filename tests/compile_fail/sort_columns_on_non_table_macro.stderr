error: `#[sort_columns]` only works on the `diesel::table!` macro
  --> $DIR/sort_columns_on_non_table_macro.rs:14:1
   |
14 | foo! {}
   | ^^^
